docs/source/getting_started/active_learning.rst

```diff
@@ -101,23 +101,23 @@ Then, in your Python script, you will need to initialize the `ApiWorkflowClient`
    and tell the `ActiveLearningAgent` to only sample from this tag. To do so, set
    the `query_tag_name` argument in the constructor of the agent.
 
-Let's configure the sampling request and request an initial selection next:
+Let's configure the selection request and request an initial selection next:
 
 .. code-block:: Python
 
-   from lightly.active_learning.config import SamplerConfig
+   from lightly.active_learning.config import SelectionConfig
    from lightly.openapi_generated.swagger_client import SamplingMethod
 
    # we want an initial pool of 150 images
-   config = SamplerConfig(n_samples=150, method=SamplingMethod.CORESET, name='initial-selection')
+   config = SelectionConfig(n_samples=150, method=SamplingMethod.CORESET, name='initial-selection')
```

Comment:
this is now really ugly because it's inconsistent :/ we always talk about selection strategy and all of the sudden it's `method=SamplingMethod.CORESET`

Reply:
This will need to be changed with https://linear.app/lightly/issue/LIG-614/rename-sampling-to-selection-internal-in-pip-package-code